doc/source/upgrading.md

```diff
@@ -1,33 +1,78 @@
-# Upgrading The helm chart
+# Upgrading the helm chart
 
-## General upgrade advice
+## General upgrade tips
 
-In general, it should be safe to upgrade a minor revision of the chart to
-receive bugfixes and improvements. A major chart upgrade (e.g. v0.4 to v0.5)
-can be more challenging, but is possible. We still recommend a full fresh
-installation if you can, but recognize it is not possible all the time. We've
-provided and tested the following upgrade instructions. If you are planning an
-upgrade of a critical major installation, we recommend you test it out on a
-staging cluster first before applying it to production. Feel free to reach out
-to us on [gitter](http://gitter.im/jupyterhub/jupyterhub) or the [mailing
-list](https://groups.google.com/forum/#!forum/jupyter) for upgrade help!
+- We **recommend** a full fresh installation instead of upgrade if you can, 
```

Comment:
I don't think we recommend this in general, only for specific cases (such as for 0.4 - 0.5). I don't think we should mark this as a general recommendation.

Reply:
I think I'll actually just send a followup PR instead of nitpicking here! Didn't realize it got merged :)

Reply:
Makes sense.